docs/core/tools/local-tools-how-to-use.md

```diff
@@ -0,0 +1,191 @@
+
+Open the *.config/dotnet-tools.json* file, and replace the contents with the following JSON:
+
+```json
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "botsay": {
+      "version": "1.0.0",
+      "commands": [
+        "botsay"
+      ]
+    },
+    "dotnetsay": {
+      "version": "2.1.3",
+      "commands": [
+        "dotnetsay"
+      ]
+    }
+  }
+}
+```
+
+Making this change is the same as getting the latest version from the repository after someone else installed the package `dotnetsay` for the project directory. 
+
+Run the `dotnet tool restore` command.
+
+```dotnetcli
+dotnet tool restore
+```
+
+The command produces output like the following example:
+
+```console
+Tool 'botsay' (version '1.0.0') was restored. Available commands: botsay
+Tool 'dotnetsay' (version '2.1.3') was restored. Available commands: dotnetsay
+Restore was successful.
+```
+
+Verify that the tools are available:
+
+```dotnetcli
+dotnet tool list
+```
+
+The output is a list of packages and commands, similar to the following example:
+
+```console
+Package Id      Version      Commands       Manifest
+------------------------------------------------------------------------------------------------------
+botsay          1.0.0        botsay         C:\Users\username\Documents\botsay\.config\dotnet-tools.json
+dotnetsay       2.1.3        dotnetsay      C:\Users\username\Documents\botsay\.config\dotnet-tools.json
+```
+
+Test the tools:
+
+```dotnetcli
+dotnet tool run dotnetsay hello from dotnetsay
+dotnet tool run botsay hello from botsay
+```
+
+## Update a local tool
+
+The installed version of local tool `dotnetsay` is 2.1.3.  The latest version is 2.1.4. Use the [dotnet tool update](dotnet-tool-update.md) command to update the tool to the latest version.
+
+```dotnetcli
+dotnet tool update dotnetsay
+```
+
+The output indicates the new version number:
+
+```console
+Tool 'dotnetsay' was successfully updated from version '2.1.3' to version '2.1.4'
+(manifest file C:\Users\username\Documents\botsay\.config\dotnet-tools.json).
+```
+
+The update command finds the first manifest file that contains the package ID and updates it. If there is no such package ID in any manifest file that is in the scope of the search, the SDK adds a new entry to the closest manifest file. The search scope is up through parent directories until a manifest file with `isRoot = true` is found.
+
+## Remove local tools
+
+Remove the installed tools by running the [dotnet tool uninstall](dotnet-tool-uninstall.md) command:
+
+```dotnetcli
+dotnet tool uninstall botsay
+dotnet tool uninstall dotnetsay
+```
+
+## Troubleshooting
+
+If you get an error message while following the tutorial, see [Troubleshoot .NET Core tool usage issues](troubleshoot-usage-issues.md)
+
+## Next steps
```

Comment:
```suggestion
## See also
```

Reply:
Seems weird to put the overview as a next step since it comes first in the TOC and it's an overview.

Reply:
Changed the name of the overview to how to.   It does have info not covered in the tutorials.